packages/mdc-snackbar/mdc-snackbar.scss

```diff
@@ -34,7 +34,7 @@
   @include mdc-snackbar-z-index($mdc-snackbar-z-index);
   @include mdc-snackbar-viewport-margin($mdc-snackbar-viewport-margin-narrow);
 
-  display: flex;
+  display: none;
```

Comment:
You should be able to clean out the display properties on the individual elements inside the snackbar now, right?

Reply:
Done